pkg/generate/code/check.go

```diff
@@ -14,14 +14,16 @@
 package code
 
 import (
+	"errors"
 	"fmt"
 	"strings"
 
-	awssdkmodel "github.com/aws/aws-sdk-go/private/model/api"
-
-	ackgenconfig "github.com/aws-controllers-k8s/code-generator/pkg/generate/config"
 	"github.com/aws-controllers-k8s/code-generator/pkg/model"
 	"github.com/aws-controllers-k8s/code-generator/pkg/names"
+	"github.com/aws-controllers-k8s/code-generator/pkg/util"
+
+	ackgenconfig "github.com/aws-controllers-k8s/code-generator/pkg/generate/config"
+	awssdkmodel "github.com/aws/aws-sdk-go/private/model/api"
```

Comment:
nit: The standard library packages are always in the first group, external modules are in the second one and internal imports are in the third group.